src/util/indent/lexi.h

```diff
@@ -0,0 +1,14 @@
+enum rwcodes {
```

Comment:
Would you please remove the enum from the c file, so we don't have a duplicate definition of the enum?